PRIVACY_AND_SECURITY.md

```diff
@@ -0,0 +1,82 @@
+# Answers to [Security and Privacy Questionnaire](https://www.w3.org/TR/security-privacy-questionnaire/)
+
+### 3.1 Does this specification deal with personally-identifiable information?
+
+No.
+
+
+### 3.2 Does this specification deal with high-value data?
+
+No.
+
+
+### 3.3 Does this specification introduce new state for an origin that persists across browsing sessions?
+
+No.
+
+
+### 3.4 Does this specification expose persistent, cross-origin state to the web?
+
+No.
+
+### 3.5 Does this specification expose any other data to an origin that it doesn’t currently have access to?
+
+Kind of. Origins will explicitly be told how they have been launched. Previously, they would have had to infer this based on requests (though this information **WAS** previously inferrible).
```

Comment:
Could you be a bit more concrete on what they will get access to that they didn't previously? (Or how the information was inferred previously?)

Reply:
Done.